spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,267 @@
+'use strict'
+
+function resetItems(){
+  items = []
+}
+
+function populateItems(){
+  items.push(new Item('+5 Dexterity Vest', 10, 20));
```

Comment:
More than one item under test can lead to confusion and breaking tests when you don't expect it - this also introduces a dependency between the tests which is not something you want.